pkg/test/fuzz/generate.go

```diff
@@ -312,6 +319,15 @@ func Visit(msgPath string, msg protoreflect.Message, setter func(v protoreflect.
 					visitor.VisitPrimitive(path+"[]", el, setter)
 				}
 
+			case protoreflect.BytesKind:
```

Comment:
Defer to the question about if we should change []byte to string, we may not want this type extension